modules/cert-manager-prepare-cloud-credentials-aws-non-sts.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-prepare-cloud-credentials-aws-non-sts"]
+= Preparing cloud credentials for the {cert-manager-operator}
+
+You can prepare cloud credentials for the {cert-manager-operator} on the AWS Security Token Service (STS) cluster by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+.Procedure
+
+. Create the `cert-manager` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace cert-manager
+----
+
+. create a directory to store the `CredentialsRequest` custom resource (CR) of the {cert-manager-operator} by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Change the directory by running the following command:
++
+[source,terminal]
+----
+$ cd credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "route53:GetChange"
+      effect: Allow
+      resource: "arn:aws:route53:::change/*"
+    - action:
+      - "route53:ChangeResourceRecordSets",
+      - "route53:ListResourceRecordSets"
+      effect: Allow
+      resource: "arn:aws:route53:::hostedzone/*"
+    - action:
+      - "route53:ListHostedZonesByName"
+      effect: Allow
+      resource: "*"
+  secretRef:
+    name: aws-creds
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
+
+. Create a `CredentialsRequest` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sample-credential-request.yaml
+----
+
+. Update the subscription object for {cert-manager-operator} by running the following command:
++
+[source,terminal]
+----
+$ oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"CLOUD_CREDENTIALS_SECRET_NAME","value":"aws-creds"}]}}}'
```

Comment:
We need to tweak the value of the subscription to use: "openshift-cert-manager-operator" instead of "cert-manager-operator".
xref: https://github.com/openshift/openshift-docs/pull/59777#discussion_r1196215594